packages/wonder-blocks-modal/components/standard-modal.test.js

```diff
@@ -0,0 +1,23 @@
+// @flow
+import * as React from "react";
+import {shallow} from "enzyme";
+
+import StandardModal from "./standard-modal.js";
+
+describe("StandardModal", () => {
+    test("Clicking the close button triggers `onClickCloseButton`", () => {
```

Comment:
We should probably do a full render so that we can check the whole setup, i.e. does clicking the close button on the modal dismiss it.  Since this button is only connected in the next diff, maybe update these tests in the next diff to do a full render and check that the modal is actually dismissed.

Reply:
I wonder if maybe we should keep these unit tests, but also write a top-to-bottom integration test separately? I feel like they serve different purposes.

Reply:
Punting integration tests to another time.